cppcodec/detail/stream_codec.hpp

```diff
@@ -241,7 +241,7 @@ struct alphabet_index_info
     static constexpr const alphabet_index_t padding_idx = 1 << 8;
     static constexpr const alphabet_index_t invalid_idx = 1 << 9;
     static constexpr const alphabet_index_t eof_idx = 1 << 10;
-    static constexpr const alphabet_index_t stop_character_mask = 0xFF00;
+    static constexpr const alphabet_index_t stop_character_mask{~0xFFu};
```

Comment:
Hm, this appears to fix MSVC performance but Clang is still lagging behind the previous state? Need to test on my local Clang how it behaves with various syntax options, and figure out how to write it so there's no performance regression.

Reply:
Confirmed locally that this version vs. the known-good version from https://travis-ci.org/tplgy/cppcodec/jobs/375293270 perform the same on Clang 6.0 (Release).

Via https://travis-ci.org/jpetso/cppcodec/jobs/404964430 I verified that Clang 6.0 (Release) performs similar to Clang 5.0 (Release). So it seems fair to blame the "regression" in Clang performance on internal differences in Travis CI such as getting a different machine assigned or a higher share of CPU cycles vs. other VMs on the same host.

I'll regard this as a fix and this merge as the new baseline.